packages/siwx/src/configs/CloudAuthSIWX.ts

```diff
@@ -72,7 +72,10 @@ export class CloudAuthSIWX implements SIWXConfig {
 
       const siweCaipNetworkId = `eip155:${siweSession?.chainId}`
```

Comment:
@tomiir I remember we've been checking this code piece together and refactored xd Not sure if it's broken later or we introduced an issue